eclipse-orion/orion.conf

```diff
@@ -0,0 +1,115 @@
+#orion.file.allowedPaths=/home/orion
+#orion.file.allowedPaths=C:\\workspaces\\orion
+
+#uncomment out this line to enable anonymous read access to files in your launched server
+#orion.file.anonymous.read=true
+
+#uncomment this line to configure virtual hosts used for site launching.
+#orion.site.virtualHosts=127.0.0.1,127.0.0.2
+#orion.site.virtualHosts=http://*.orionhub.org:8080
+
+#uncomment this line and set a password to create an admin user
+#orion.auth.admin.default.password={password}
+
+#uncomment this line to enable the log provider support
+#orion.logs.logProviderEnabled=true
+
+#uncomment this line to provide custom log buffer size used while downloading log files.
+#The default value is 64 KB, the buffer size is given in bytes.
+#orion.logs.logBufferSize=65536
+
+#uncomment the lines below and specify the keystore location and passwords to launch your server over https
+#jetty.https.enabled=true
+#jetty.https.port=443
+#jetty.ssl.keystore={keystore_location}
+#jetty.ssl.password={password}
+#jetty.ssl.keypassword={password}
+#jetty.ssl.protocol=SSLv3
+
+#uncomment and update these lines to configure the smtp host for sending mail
+#mail.smtp.host={smtp_host}
+#mail.smtp.port=25
+#mail.smtp.user={smtp_user}
+#mail.smtp.password={password}
+#mail.smtp.auth=true
+#mail.smtp.starttls.enable=true
+#mail.from={sender_of_mails}
+#mail.debug=false
+
+#uncomment this line to specify the username for the MQTT message broker
+#orion.events.user={username}
+
+#uncomment this line to specify the password for the MQTT message broker
+#orion.events.password={password}
+
+#uncomment this line to specify the URI for the MQTT message broker
+#orion.events.uri=tcp://localhost:1883
+
+#uncomment this line to specify a clientId for this server instance when reporting events to the MQTT message broker
+#orion.events.clientId={clientId}
+
+#uncomment this line to enable access logs in NCSA format on the Jetty server
+#orion.jetty.access.logs.enable=true
+
+#uncomment and update these lines and specify the oauth client and secret keys
+#orion.oauth.google.client={client_key}
+#orion.oauth.google.secret={secret_key}
+#orion.oauth.github.client={client_key}
+#orion.oauth.github.secret={secret_key}
+
+#uncomment this line to enable cross-site request forgery protection
+#orion.XSRFPreventionFilterEnabled=true
+
+#uncomment and update this line to enable client logging to a Google Analytics account
+#orion.metrics.google.tid={tid}
+
+#uncomment and update this line to set the site speed sampling rate (0-100) when logging to a Google Analytics account
+#orion.metrics.google.sitespeed.sample={rate}
+
+#uncomment and update this line to set the context path to the Orion servlets. The default value is /
+#indicating that Orion is running at the server root.
+#orion.context.path=/
+
+#uncomment this line to enable the cf liveupdate feature.
+orion.cf.liveupdate.enabled=true
```

Comment:
We should disable this. https://appsembler.slack.com/archives/G03M2HMCC/p1502119916016045